packages/bits/src/lib/table/table.component.less

```diff
@@ -87,8 +87,6 @@
 
         &.cdk-table-sticky { // Border hack because position sticky collapses borders and they are not visible
             border: 0;
-            box-shadow: 5px 0 5px 0 @nui-shadow-color;
-            box-shadow: 5px 0 5px 0 var(--nui-shadow-color,@nui-shadow-color);
```

Comment:
Remove unneeded box-shadows. The box shadows are usually hidden from view, but Firefox reveals the far right header cell box shadow when the window is resized.

![firefox box shadow](https://user-images.githubusercontent.com/10159693/113359223-f426b480-930c-11eb-9774-ae06b8f04b5e.gif)